.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
Can you remove these additional files in the `.idea/` directory? I don't think they're necessary to make this change!

Reply:
Addressed in #764